lib/con_cache.ex

```diff
@@ -136,6 +136,16 @@ defmodule ConCache do
   defp validate_ttl(_ttl_check_interval, nil), do: raise ArgumentError, "ConCache global_ttl must be supplied"
   defp validate_ttl(_ttl_check_interval, _global_ttl), do: :ok
 
+  @spec child_spec([options]) :: Supervisor.child_spec()
```

Comment:
I can't figure out how to create `@spec` with 2 lists (for `start_link` attrs), could you help me with that?